src/hooks/useColor.ts

```diff
@@ -1,46 +1,63 @@
-import { useState, useLayoutEffect } from 'react'
-import { shade } from 'polished'
-import Vibrant from 'node-vibrant'
-import { hex } from 'wcag-contrast'
 import { Token } from '@uniswap/sdk-core'
+import Vibrant from 'node-vibrant'
+import { shade } from 'polished'
+import { useLayoutEffect, useState } from 'react'
+import { WrappedTokenInfo } from 'state/lists/wrappedTokenInfo'
 import uriToHttp from 'utils/uriToHttp'
+import { hex } from 'wcag-contrast'
+
+function URIForEthToken(address: string) {
+  return `https://raw.githubusercontent.com/uniswap/assets/master/blockchains/ethereum/assets/${address}/logo.png`
+}
 
 async function getColorFromToken(token: Token): Promise<string | null> {
-  if (token.chainId !== 1) {
-    return Promise.resolve('#FAAB14')
+  if (!(token instanceof WrappedTokenInfo)) {
+    return null
   }
 
-  const path = `https://raw.githubusercontent.com/uniswap/assets/master/blockchains/ethereum/assets/${token.address}/logo.png`
-
-  return Vibrant.from(path)
-    .getPalette()
-    .then((palette) => {
-      if (palette?.Vibrant) {
-        let detectedHex = palette.Vibrant.hex
-        let AAscore = hex(detectedHex, '#FFF')
-        while (AAscore < 3) {
-          detectedHex = shade(0.005, detectedHex)
-          AAscore = hex(detectedHex, '#FFF')
-        }
-        return detectedHex
-      }
+  const wrappedToken = token as WrappedTokenInfo
+  const { address } = wrappedToken
+  let { logoURI } = wrappedToken
+  if (!logoURI) {
+    if (token.chainId !== 1) {
```

Comment:
i know that what we had before but probably worth updating to `SupportedChainId.MAINNET`